nearpy/storage/storage_mongo.py

```diff
@@ -159,14 +178,14 @@ def clean_buckets(self, hash_name):
         """
         Removes all buckets and their content for specified hash.
         """
-        self.mongo_object.remove(
+        self.mongo_object.delete_many(
             {'lsh': {'$regex': self._format_hash_prefix(hash_name)}})
 
     def clean_all_buckets(self):
         """
         Removes all buckets from all hashes and their content.
         """
-        self.mongo_object.remove(
+        self.mongo_object.delete_many(
```

Comment:
`remove` method is deprecated so I replaced it with the suggested `delete_many`. It avoids annoying deprecation warnings